src/app/pages/home/services/home.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@angular/core'
+import { UsageEventType } from '../../../shared/enums/events'
+import { UsageService } from '../../../core/services/usage/usage.service'
+
+@Injectable()
+export class HomeService {
```

Comment:
This methods in this service seems to just obfuscate calls to the `UsageService`, is it really needed?

Reply:
Yup, not needed. Removed this.